android/src/main/java/com/arlocalizerrn/ArActivity.kt

```diff
@@ -12,27 +12,49 @@ import com.netguru.arlocalizerview.ARLocalizerDependencyProvider
 import com.netguru.arlocalizerview.arview.ARLocalizerView
 import com.netguru.arlocalizerview.location.LocationData
 
+
 class ArActivity : AppCompatActivity(), ARLocalizerDependencyProvider {
 
-  override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
+    private fun getPoints():MutableList<LocationData>{
+        val locationData = intent.getSerializableExtra("locations") as? LocationList
+        val points = mutableListOf<LocationData>()
+
+        val locations = locationData?.locations
+
+        if (locations != null) {
+            for (location in locations) {
+                points.add(LocationData(location["latitude"] as Double, location["longitude"] as Double))
+            }
+        }
+        return points
+
+    }
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+
+
+
+
```

Comment:
These spaces here are required ? I'm not sure if github display this file right way :)

Reply:
done